content/docs/nav.yml

```diff
@@ -1,85 +1,85 @@
-- title: Installation
+- title: Kurulum
   items:
     - id: getting-started
-      title: Getting Started
+      title: Başlangıç
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: Bir Web Sitesine React Eklemek
     - id: create-a-new-react-app
-      title: Create a New React App
+      title: Yeni bir React Uygulaması Oluşturun
     - id: cdn-links
-      title: CDN Links
-- title: Main Concepts
+      title: CDN Bağlantıları
+- title: Temel Kavramlar
   isOrdered: true
   items:
     - id: hello-world
-      title: Hello World
+      title: Merhaba Dünya
     - id: introducing-jsx
-      title: Introducing JSX
+      title: JSX'e Giriş
     - id: rendering-elements
-      title: Rendering Elements
+      title: Elementlerin Render Edilmesi
     - id: components-and-props
-      title: Components and Props
+      title: Bileşenler ve Prop'lar
     - id: state-and-lifecycle
-      title: State and Lifecycle
+      title: State ve Yaşam Döngüsü
     - id: handling-events
-      title: Handling Events
+      title: Olay Yönetimi
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: Koşullu Renderlama
     - id: lists-and-keys
-      title: Lists and Keys
+      title: Listeler ve Anahtarlar
     - id: forms
-      title: Forms
+      title: Formlar
     - id: lifting-state-up
-      title: Lifting State Up
+      title: State'i Yukarı Taşıma
     - id: composition-vs-inheritance
-      title: Composition vs Inheritance
+      title: Bileşim vs Kalıtım
     - id: thinking-in-react
-      title: Thinking In React
-- title: Advanced Guides
+      title: React'te Düşünmek
+- title: Gelişmiş Kılavuz
   items:
     - id: accessibility
-      title: Accessibility
+      title: Erişebilirlik
     - id: code-splitting
-      title: Code-Splitting
+      title: Kod Bölümleme
     - id: context
-      title: Context
+      title: Bağlam (Context)
     - id: error-boundaries
-      title: Error Boundaries
+      title: Hata Sınırları
     - id: forwarding-refs
-      title: Forwarding Refs
+      title: Ref'leri Taşımak
     - id: fragments
-      title: Fragments
+      title: Fragment'ler
     - id: higher-order-components
-      title: Higher-Order Components
+      title: Üst-Seviye Bileşenler
     - id: integrating-with-other-libraries
-      title: Integrating with Other Libraries
+      title: Başka Kütüphaneleri Entegre Etmek
     - id: jsx-in-depth
-      title: JSX In Depth
+      title: Derinlemesine JSX
     - id: optimizing-performance
-      title: Optimizing Performance
+      title: Performansı En İyi Duruma Getirmek
     - id: portals
-      title: Portals
+      title: Portal'lar
     - id: react-without-es6
-      title: React Without ES6
+      title: ES6 Olmadan React Kullanımı
     - id: react-without-jsx
-      title: React Without JSX
+      title: JSX Olmadan React Kullanımı
     - id: reconciliation
       title: Reconciliation
```

Comment:
Proposal expecting

Reply:
> [Uyumlaştırma](https://github.com/reactjs/tr.reactjs.org/blob/afa116d2cc517f125522721ed654c54764b90fe0/content/docs/reference-glossary.md#uyumla%C5%9Ft%C4%B1rma-reconciliation)

This is a great translation by @suyalcinkaya. We should use it.

Reply:
@alperen Oops. Forgot to include this into my review.